packages/iov-ethereum/src/erc20.ts

```diff
@@ -9,3 +21,5 @@ export interface Erc20Options {
   /** Override on-chain name. Use this if contract does not define value on-chain. */
   readonly name?: string;
 }
+
+export type Erc20TokensMap = ReadonlyMap<TokenTicker, Erc20Options>;
```

Comment:
Since this type is part of the @iov/ethereum public interface, it should be exported in `index.ts`